daprdocs/content/en/dotnet-sdk-docs/dotnet-development/_index.md

```diff
@@ -0,0 +1,181 @@
+---
+type: docs
+title: "Dapr .NET SDK Development"
+linkTitle: "Development"
+weight: 1000
```

Comment:
```suggestion
weight: 50000
```

Reply:
This will have the left sidebar go:

- Client
- Server
- Actors
- Development
- Troubleshooting